pkgs/data/fonts/hyperscrypt/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchzip, lib }:
+
+stdenv.mkDerivation rec {
+
+  pname = "HyperScrypt";
+  version = "1.1";
+  name = "${lib.toLower pname}-font-${version}";
+
+src = fetchzip {
+  url = "https://gitlab.com/StudioTriple/Hyper-Scrypt/-/archive/${version}/Hyper-Scrypt-${version}.zip";
+  sha256 = "1swxk5qhr3g9l8kkbpahx7l15f8jk1q8i6k35ax846dzs03l1n9q";
+};
+
+installPhase = ''
```

Comment:
This ```installPhase``` could be ```postFetch``` in ```fetchzip```, so the whole derivation will be fixed-output and the will be a single binary for all platforms (see others fonts as example)